deepchem/models/torch_models/smiles2vec.py

```diff
@@ -177,3 +182,103 @@ def forward(self, smiles_seqs: List):
         else:
             Logits = Logits.view(-1, self.n_tasks, 1)
             return Logits
+
+
+class Smiles2VecModel(TorchModel):
```

Comment:
Please doublecheck it is exactly the same name as TF. Also make sure all the arguments overlap

Reply:
checked them, and they do overlap